qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -0,0 +1,582 @@
+        """
+        if qargs is None:
+            qubits = range(self.clifford.num_qubits)
+        else:
+            qubits = qargs
+
+        outcome = ["X"] * len(qubits)
+        outcome_prob = 1.0
+        probs = {}  # probabilities dictionary
+
+        self._get_probablities(qubits, outcome, outcome_prob, probs)
+
+        if decimals is not None:
+            for key, value in probs.items():
+                probs[key] = round(value, decimals)
+
+        return probs
+
+    def reset(self, qargs=None):
+        """Reset state or subsystems to the 0-state.
+
+        Args:
+            qargs (list or None): subsystems to reset, if None all
+                                  subsystems will be reset to their 0-state
+                                  (Default: None).
+
+        Returns:
+            StabilizerState: the reset state.
+
+        Additional Information:
+            If all subsystems are reset this will return the ground state
+            on all subsystems. If only some subsystems are reset this
+            function will perform a measurement on those subsystems and
+            evolve the subsystems so that the collapsed post-measurement
+            states are rotated to the 0-state. The RNG seed for this
+            sampling can be set using the :meth:`seed` method.
+        """
+        # Resetting all qubits does not require sampling or RNG
+        if qargs is None:
+            return StabilizerState(Clifford((np.eye(2 * self.clifford.num_qubits))))
+
+        ret = self.copy()
+        for qubit in qargs:
+            # Apply measurement and get classical outcome
+            outcome = ret._measure_and_update(qubit, 0)
+
+            # Use the outcome to apply X gate to any qubits left in the
+            # |1> state after measure, then discard outcome.
+            if outcome == 1:
+                _append_x(ret.clifford, qubit)
+
+        return ret
+
+    def measure(self, qargs=None):
+        """Measure subsystems and return outcome and post-measure state.
+
+        Note that this function uses the QuantumStates internal random
+        number generator for sampling the measurement outcome. The RNG
+        seed can be set using the :meth:`seed` method.
+
+        Args:
+            qargs (list or None): subsystems to sample measurements for,
+                                  if None sample measurement of all
+                                  subsystems (Default: None).
+
+        Returns:
+            tuple: the pair ``(outcome, state)`` where ``outcome`` is the
+                   measurement outcome string label, and ``state`` is the
+                   collapsed post-measurement stabilizer state for the
+                   corresponding outcome.
+        """
+        if qargs is None:
+            qargs = range(self.clifford.num_qubits)
+
+        ret = self.copy()
+        outcome = ""
+        for qubit in qargs:
+            randbit = ret._rng.randint(2)
```

Comment:
This should raise an error because `_rng` is a `Generator` object. It should be:
```suggestion
            randbit = ret._rng.integers(2)
```
I checked the base `QuantumState` class and it looks like there is a bug in the `def _rng` property. It should be:
```python
@property
    def _rng(self):
        if self._rng_generator is None:
            return np.random.default_rng()
        return self._rng_generator
```